test/attach_detach_test_v2.sh

```diff
@@ -0,0 +1,181 @@
+# use sh attach_detach_test.sh 500 default to test 500 pods.
```

Comment:
delete this attach_detach_test_v2.sh script?

Reply:
ok I fix the comment.